web-module/pom.xml

```diff
@@ -56,7 +56,7 @@
     <dependency>
       <groupId>com.celements</groupId>
       <artifactId>celements-blog</artifactId>
-      <version>1.45</version>
+      <version>1.45-SNAPSHOT</version>
```

Comment:
you should not change from a released version to a snapshot. More likely the intended version is 1.46-SNAPSHOT